content/en/docs/ambient/usage/extend-waypoint-wasm/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Distributing WebAssembly Modules
+description: Describes how to make remote WebAssembly modules available for ambient mode.
+weight: 45
+keywords: [extensibility,Wasm,WebAssembly,Ambient]
+owner: istio/wg-policies-and-telemetry-maintainers
```

Comment:
an odd owner

Reply:
All wasm documents are currently under this ownership, not sure why.. so did not change.